src/comm.F

```diff
@@ -257,7 +257,7 @@ subroutine getcorner_GPU(s,nw1,nw2,ne1,ne2,sw1,sw2,se1,se2)
       integer reqs(8)
       integer tag1,tag2,tag3,tag4
       integer, dimension(mpi_status_size,8) :: status1
-      logical, parameter :: Debug =.TRUE.
+      logical, parameter :: Debug =.FALSE.
```

Comment:
Can we move `Debug` parameter to a shared module so that we only need to change its value once?

Reply:
I added the Debug flag to turn on print statements when I was looking at issues with the GPU-direct.  I don't envision that this variable will be around long term, just until the code is fully debugged.

Reply:
I see. Then we could clean it up when the code is in a good shape.